### DISCUSSION TITLE
Welcome to lazylualinq Discussions!

### DISCUSSION THREAD

**Discussion by Henkoglobin** [Henkoglobin/lazylualinq#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Henkoglobin** [comment]
Hi, I'm Henkoglobin! I started lazylualinq in college as a part of my bachelor's thesis and continue using it to this day. 

If you're using lazylualinq, I'd be delighted to hear about your project and how lazylualinq could be better! Are there any rough edges you've encountered? Operators you were missing? Cool things it helped you with? Please feel free to share!

你好！ 遗憾的是我不会说中文，但我看到一些来自中国的人使用lazylualinq - 谢谢！ 如果您正在使用lazylualinq，请随时告诉我们，我很想听听！

附言 我希望谷歌翻译不要把这个翻译搞砸！

**Comment by svermeulen** [comment]
Hello!  I make heavy use of lazylualinq inside my neovim config, and also some of the neovim plugins I work on.  I'm also working on a game in Rust that uses Lua as a scripting language, and am using lazylualinq there as well.
